src/main/java/jalau/at18/katas/pokerhands/sarai/CardValuesCount.java

```diff
@@ -4,18 +4,18 @@
 
 public class CardValuesCount {
 
-    private HashMap<CardValue, Integer> valuesCount;
+    private HashMap<CardValue, Integer> valuesCount; //listas dinamicas  key / valor
 
     public CardValuesCount() {
         valuesCount = new HashMap<CardValue, Integer>();
     }
 
     public void add(CardValue cardValue) {
         int existingCount = 0;
-        if (valuesCount.containsKey(cardValue)) {
+        if (valuesCount.containsKey(cardValue)) { // valores de la carta
             existingCount = get(cardValue);
         }
-        valuesCount.put(cardValue, existingCount + 1);
+        valuesCount.put(cardValue, existingCount + 1); //carta enviada
```

Comment:
even for comments, lets use english. Or if this was not meant to be commited, you can remove it